libs/SmartStore/SmartStore/Classes/SFSmartStoreUpgrade.h

```diff
@@ -24,12 +24,14 @@
 
 #import <Foundation/Foundation.h>
 #import <SalesforceSDKCore/SFUserAccount.h>
+#import <SalesforceSDKCore/SalesforceSDKConstants.h>
 
 NS_ASSUME_NONNULL_BEGIN
 
 /**
  Used internally for upgrading SmartStore.
  */
+SFSDK_DEPRECATED(8.3, 9.0, "Will be removed.")
```

Comment:
The upgrade steps are from 2014 except for `updateEncryptionSalt` which is from 7.1, if there's any reason to keep that method around longer I'm thinking the class could still be deprecated and be internal